UnityServer/.firebaserc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should we add `.firebaserc` files to .gitignore?

Reply:
Yep, going to add it

Reply:
should we delete this file and not commit it now that it's in .gitignore?

Reply:
Did not see the message, sorry. Will do